### ISSUE TITLE
Feature Request: Component for reassigning VRC humanoid Colliders

### ISSUE THREAD

**Issue by ZenithVal** [bdunderscore/modular-avatar#688]
The hand and finger colliders on your avatar that can interact with other players' physbones can be re-assigned if you open the descriptor component in inspector debug. One popular gimmick is assigning the ring finger bone to the foot, so they can interact with physbones using it. 

D4rk's optimizer has a simple toggle for assigning the ring finger to the foot collider. [Link]( https://github.com/d4rkc0d3r/d4rkAvatarOptimizer/blob/6f8a76985b4783f06e6cf21102a0dfe6bffb11bc/Editor/d4rkAvatarOptimizer.cs#L4523)

The only issue is that if you exit debug mode and the collier section is still open, the SDK will revert these bones to their humanoid default. D4rk's optimizer gets around this by applying it with a script without ever opening the descriptor collider section. (Non destructively too) 

Other example use cases:
 - Colliders for props so they affect Physbones.
 - Assigning the collider to a constraint so you can move it anywhere on a whim.
   - I toggle my hand between my mouth and hand ~ allowing me to grab physbones with it (Bite!)

Basically, a component for full custom assignments of the colliders. Adjusting their size with a preview may also be appreciated since that's not possible in debug mode.
